.gitignore

```diff
@@ -108,3 +108,6 @@ dist/
 # Patch files
 *.patch
 scratch/
+
+# Nexus
+nexus-data
```

Comment:
I had to set `user: <vagrant uid>` to avoid a permissions issue that blocked the server from starting in my Vagrant VM.